docs/ff-concepts/adding-customization/code-file.md

```diff
@@ -0,0 +1,419 @@
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/dVK7pbda7U6B4igmNUP7?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+## Using Static Classes
+
+Sometimes, you just want to run a small piece of logic like formatting text or doing a calculation, without needing to set up anything extra. That’s when `static` data members are helpful. You can access static fields and methods without creating an instance of a class. You can think of them like tools that are always ready to use.
+
+This approach is typically used for **stateless utility classes** where shared functionality is needed across the app. For example, look at the class below:
+
+```jsx
+class Utils {
+  static int square(int x) => x * x;
+}
+```
+
+The `Utils` class contains a static method `square` that returns the square of a number without needing to create an object of the class.
+
+Here are couple more examples to understand it better:
+
+- This `StringFormatter` class below provides reusable static methods to capitalize text, convert it to lowercase, or format it in snake_case.
+
+  ```jsx
+  class StringFormatter {
+  static String lastFormatted = '';
+  static int formatCount = 0;
+  
+  static String capitalize(String input) =>
+      input[0].toUpperCase() + input.substring(1);
+
+  static String toLowerCase(String input) => input.toLowerCase();
+
+  static String toSnakeCase(String input) =>
+      input.replaceAll(' ', '_').toLowerCase();
+  }
+  ```
+
+- The `MathHelper` class offers handy static methods to calculate tax, apply discounts, find percentages, and round off numbers.
+
+  ```jsx
+  class MathHelper {
+    static double calculateTax(double amount) => amount * 0.18;
+
+    static double applyDiscount(double amount, double discountPercent) =>
+        amount - (amount * discountPercent / 100);
+
+    static double calculatePercentage(double part, double total) =>
+        (part / total) * 100;
+
+    static int roundOff(double value) => value.round();
+  }
+  ```
+
+:::warning
+
+Static classes are powerful, but they should be used carefully. Overusing static methods can lead to less flexible code and potential issues, especially when the logic requires access to state or needs to evolve over time. Stick to static methods only when the logic is truly independent and doesn’t rely on instance-specific data.
```

Comment:
can fix the first line based on the feedback about "static classes"

Reply:
also mention somewhere that static and instance fields can be mixed in a class as well?